src/code/z_camera.c

```diff
@@ -1107,11 +1107,12 @@ s32 Camera_CalcAtForLockOn(Camera* camera, VecGeo* eyeAtDir, Vec3f* targetPos, f
     f32 playerHeight;
 
     playerHeight = Player_GetHeight(camera->player);
-    tmpPos0.x = 0.0f;
-    tmpPos0.y = playerHeight + yOffset;
-    tmpPos0.z = 0.0f;
-    if (PREG(76) && (flags & FLG_ADJSLOPE)) {
-        tmpPos0.y -= Camera_CalcSlopeYAdj(floorNorm, playerPosRot->rot.y, eyeAtDir->yaw, R_CAM_SLOPE_Y_ADJ_AMOUNT);
+    playerToAtOffsetTarget.x = 0.0f;
+    playerToAtOffsetTarget.y = playerHeight + yOffset;
+    playerToAtOffsetTarget.z = 0.0f;
+    if (R_CAM_PARALLEL_LOCKON_CALC_SLOPE_Y_ADJ && (flags & CAM_LOCKON_AT_FLAG_CALC_SLOPE_Y_ADJ)) {
```

Comment:
I'm thinking of `LOCKON` -> `ZTARGET` in a different branch of mine looking at all the Camera_Calc support functions, thoughts?

Reply:
`CAM_MODE_FOLLOW_BOOMERANG` begs to differ, it seems to often use `CAM_FUNC_KEEP1`/KeepOn1 (which uses `Camera_CalcAtForLockOn`)

When looking at the level of camera behavior, it probably will look weird anyway to refer to entire functions as "this is for ztarget in the case of blah blah blah" which would be redundant with z_cam_data anyway (but can be useful for context still), it would be better to mainly describe how the behavior looks

Descriptions I expect would mainly look like "What the camera follows (eye), where the camera is placed (at), along which axes can the camera move, around which axes can the camera rotate"
I think we need much more in depth docs of every camera function before we can discuss names?

Reply:
not in a meaningful way

fwiw vanilla text equates "lock on" with z-targeting:
```
QUICKTEXT_ENABLE "In other words, if you press " COLOR(LIGHTBLUE) "[Z]" COLOR(DEFAULT) " \n"
"when you see " COLOR(ADJUSTABLE) "▼" COLOR(YELLOW) " " COLOR(DEFAULT) "above an object,\n"
"you can lock on to it." QUICKTEXT_DISABLE 
```
but that shouldn't prevent us from using "lock on" for something else

Reply:
Another challenge with this function is `playerToAtOffsetTarget`: Is the target referring to the `Camera_LERPCeilVec3f` value-target value (which it is) or is it referring to the `camera->target` which is the actor that the camera is `zTargeting`? I believe this duel usage of `target` is why there are names like `eye` and `eyeNext` instead of `eyeTarget`, because `target` could refer to either the target value for "StepTo" or the target actor in z-targeting. It's a bigger problem that extends outside this PR, but it's something to be mindful of with naming `playerToAtOffsetTarget` in a target-related function. So maybe `lockOn` could be used globally within camera to replace `targetActor`, and then `eyeNext` could be `eyeTarget` to be more consistent with decomp.

Reply:
We can also continue the LockOn/Target/ZTarget/Next discussion later since it should be a bigger discussion. For now, LOCKON is consistent with this function so it's fine